LayoutTests/fast/multicol/vertical-rl/nested-columns.html

```diff
@@ -1,18 +1,13 @@
-<html>
-<body style="width:500px; height:750px; -webkit-column-count:2; -webkit-column-fill:auto; column-count:2; column-fill:auto; -webkit-writing-mode:vertical-rl">
-
-<h1>Header One</h1>
-Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nulla varius enim ac mi. Curabitur sollicitudin felis quis lectus. Quisque adipiscing rhoncus sem. Proin nulla purus, vulputate vel, varius ut, euismod et, nisi. Sed vitae felis vel orci sagittis aliquam. Cras convallis adipiscing sem. Nam nonummy enim. Nullam bibendum lobortis neque. Vestibulum velit orci, tempus euismod, pretium quis, interdum vitae, nulla. Phasellus eget ante et tortor condimentum vestibulum.
-Suspendisse hendrerit quam nec felis. Sed varius turpis vitae pede. Lorem ipsum dolor sit amet, consectetuer adipiscing elit.
-<h1>Header Two</h1>
-Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nulla varius enim ac mi. Curabitur sollicitudin felis quis lectus. Quisque adipiscing rhoncus sem. Proin nulla purus, vulputate vel, varius ut, euismod et, nisi. Sed vitae felis vel orci sagittis aliquam. Cras convallis adipiscing sem. Nam nonummy enim. Nullam bibendum lobortis neque. Vestibulum velit orci, tempus euismod, pretium quis, interdum vitae, nulla. Phasellus eget ante et tortor condimentum vestibulum.
-Suspendisse hendrerit quam nec felis. Sed varius turpis vitae pede. Lorem ipsum dolor sit amet, consectetuer adipiscing elit.
-<h1>Header Three</h1>
-<div style="-webkit-column-count:2">
-Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nulla varius enim ac mi. Curabitur sollicitudin felis quis lectus. Quisque adipiscing rhoncus sem. Proin nulla purus, vulputate vel, varius ut, euismod et, nisi. Sed vitae felis vel orci sagittis aliquam. Cras convallis adipiscing sem. Nam nonummy enim. Nullam bibendum lobortis neque. Vestibulum velit orci, tempus euismod, pretium quis, interdum vitae, nulla. Phasellus eget ante et tortor condimentum vestibulum.
-Suspendisse hendrerit quam nec felis. Sed varius turpis vitae pede. <span style="position:relative; opacity:0.5">Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</span>
+<!DOCTYPE html>
+<p>The word 'PASSED' should be seen below (writing-mode is vertical-rl).</p>
+<div style="width:2em; height:6em; -webkit-columns:3; -webkit-column-gap:0; column-fill:auto; line-height:2em; -webkit-writing-mode:vertical-rl;">
+    <div style="-webkit-column-count:2; -webkit-column-gap:0;">
+        P<br>A
+    </div>
+    <div style="-webkit-column-count:2; -webkit-column-gap:0;">
+        S<br>S
+    </div>
+    <div style="-webkit-column-count:2; -webkit-column-gap:0;">
+        E<br>D
+    </div>
```

Comment:
I am seeing lots of red in .html files. how is this just a reftest update? (I must be missing something here)

Reply:
So Blink tweaked 'test' to get the same behavior in order to make them 'reftest' without adding too much text etc.

Reply:
@alanbaradlay - same as this - https://github.com/chromium/chromium/commit/fb77f07940e4162728766c9086d5859031ba31b8

Reply:
and we are sure about not losing coverage here.

Reply:
Yes - same test still exists on Blink as well. :-)